src/Http/ClusterTopology.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace RavenDB\Http;
+
+class ClusterTopology
+{
+    private string $lastNodeId;
+    private string $topologyId;
+    private int $etag;
+
+    // sve sto je zakomentarisano treba implementirati i kad se implementira potrebno je ukloniti iz komentara
```

Comment:
?

Reply:
translation from Serbian:

Everything that is commented out should be implemented and removed it after implementation

:D

Reply:
Class now fully implemented

Reply:
Class now fully implemented